default/python/universe_generation/galaxy.py

```diff
@@ -15,19 +15,23 @@ def __init__(self, universe_width):
         self.max_dist = universe_tables.MAX_STARLANE_LENGTH
         self.cell_size = min(max(universe_width / 50, self.min_dist), self.max_dist/sqrt(2))
         self.width = int(universe_width / self.cell_size) + 1
-        self.grid = defaultdict(list)
-        print "Adjacency Grid: width", self.width, ", cell size", self.cell_size
+        self.grid = defaultdict(set)
+        print "Adjacency Grid: width {}, cell size {}".format(self.width, self.cell_size)
 
     def cell(self, pos):
+        """Returns cell index"""
         return (int(pos[0] / self.cell_size), int(pos[1] / self.cell_size))
 
     def insert_pos(self, pos):
-        self.grid[self.cell(pos)].append(pos)
+        """Inserts pos"""
+        self.grid[self.cell(pos)].add(pos)
```

Comment:
Looks slightly wired. We have `pos` in key and in value.  If we always can recreate key by value when we don`t need dict here.    I did not dive deep into it, may be there is some reasons.

I assume that if you have getter methods for `self.grid` then you did not access it directly and can make it private by adding double underscore to its name `__`.  This really help during refactoring, because you can be sure that attribute is not used outside.